### ISSUE TITLE
Select which i2c bus to use

### ISSUE THREAD

**Issue by MarkAYoder** [threebytesfull/matrix-pi#4]
Many thanks for this code.  I have it up and running on my BeagleBone.  The only trouble I had was the code is written to use ic2 bus 1 and I'm using bus 2 on my Bone.

Code the code be modified so the i2c bus number can be passed to it?

Thanks...

--Mark

**Comment by threebytesfull** [comment]
That shouldn't be too hard - I'll have a look! I'm not sure if I have anything with a second bus so the fix might be untested... ;)

**Comment by MarkAYoder** [comment]
Great!  I'm happy to test it.

--Mark

**Comment by threebytesfull** [comment]
I've just pushed an update which adds a `-b` option to the test scripts (I'd already included bus number support in the library but haven't ever tried it!).

The web application didn't support command-line options at all so I've also added very basic support there.

`the_matrix_identify`, `the_matrix_leds`, `the_matrix_scrolltext`, `the_matrix_web` all support the `-b <bus_number>` option now. For example:

    the_matrix_web -b 2

should now open a web interface for the matrix on bus 2.

Let me know if it works :)
